packages/extension/src/background/wallet.ts

```diff
@@ -83,10 +83,8 @@ function calculateContractAddress(
   ])
 }
 
-export const equalAccount = (
-  a: Pick<WalletAccount, "address" | "network">,
-  b: Pick<WalletAccount, "address" | "network">,
-) => a.address === b.address && a.network.id === b.network.id
+export const equalAccount = (a: UniqAccount, b: UniqAccount) =>
```

Comment:
unrelated but this is better english

```suggestion
export const accountsEqual = (a: UniqAccount, b: UniqAccount) =>
```